src/simplify_reshapes.cpp

```diff
@@ -627,6 +627,30 @@ struct find_transpose_contiguous_reshaper_unary
     }
 };
 
+struct find_broadcast_transpose
+{
+    auto matcher() const
+    {
+        return match::name("transpose")(
+            match::arg(0)(match::name("multibroadcast")(match::used_once().bind("bcast_ins"))));
```

Comment:
Dont use `used_once`. `multibroadcast` is an alias operator so its cheap to run, so it ok if is done multiple times.